src/gleam/string.gleam

```diff
@@ -270,66 +370,80 @@ pub fn join(strings: List(String), with separator: String) -> String {
 /// pad_right("121", to: 2, with: ".") == "121"
 /// ```
 ///
-// pub fn pad_right(string: String, to size: Int, with: String) {}
+pub fn pad_right(pad pad: String, to_length to_length: Int, with with: String) {
+    pad_help(pad, to_length, with, Right)
+}
 
-// TODO
-/// Get rid of whitespace on both sides of a String.
+/// Remove whitespace on both sides of a String.
+/// All breaking Unicode characters with the White Space property will be removed.
+/// See: https://en.wikipedia.org/wiki/Whitespace_character#Unicode
 ///
 /// ## Examples
 /// ```gleam
 /// trim("  hats  \n") == "hats"
 /// ```
 ///
-// pub fn trim(string: String) -> String {}
+type TrimDirection {
```

Comment:
The doc comment here is attached to the type rather than the function it documents.